src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -181,37 +187,40 @@
 short-summary: Describe a helm chart in an Azure Container Registry.
 examples:
   - name: Show all versions of a helm chart in an Azure Container Registry
-    text: >
+    text: >-
         az acr helm show -n MyRegistry mychart
   - name: Show a helm chart version in an Azure Container Registry
-    text: >
+    text: >-
         az acr helm show -n MyRegistry mychart --version 0.3.2
+  - name: Describe a helm chart in an Azure Container Registry. (autogenerated)
+    text: az acr helm show --name MyRegistry mychart
+    crafted: true
 """
 
 helps['acr import'] = """
 type: command
 short-summary: Imports an image to an Azure Container Registry from another Container Registry. Import removes the need to docker pull, docker tag, docker push.
 examples:
   - name: Import an image to the target registry and inherits sourcerepository:sourcetag from the source registry.
-    text: >
+    text: >-
         az acr import -n MyRegistry --source sourceregistry.azurecr.io/sourcerepository:sourcetag
   - name: Import an image from a registry in a different subscription.
-    text: >
+    text: >-
         az acr import -n MyRegistry --source sourcerepository:sourcetag -t targetrepository:targettag -r /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/sourceResourceGroup/providers/Microsoft.ContainerRegistry/registries/sourceRegistry
   - name: Import an image from a public repository in Docker Hub
-    text: >
-        az acr import -n MyRegistry --source docker.io/library/hello-world:latest -t targetrepository:targettag
+    text: >-
+        az acr import -n MyRegistry --source docker.io/sourcerepository:sourcetag -t targetrepository:targettag
```

Comment:
Is this being updated from Aladdin?  If so, it should add the autogenerated tag to the text.

However, I believe the original example uses an actual source, whereas the replacement is non-existent placeholder. Someone from ACR can confirm.